src/client/mod.rs

```diff
@@ -263,7 +265,9 @@ impl App {
         if let Some(game_state) = game_state_option {
             self.game_state = game_state;
         } else {
-            self.game_state.update(args.dt*1000.0);
+            if let Some(ref world) = self.world_state {
+                self.game_state.update(world, args.dt*1000.0);
```

Comment:
Maybe we should always add spaces around operators?

Reply:
I think I'll run a complete rust-fmt run in a later PR and enable CI for it. But lets first merge the existing PRs.